core/src/main/resources/error/error-classes.json

```diff
@@ -998,6 +998,12 @@
     },
     "sqlState" : "42K07"
   },
+  "INVALID_SET_SYNTAX" : {
+    "message" : [
+      "Expected format is 'SET', 'SET key', or 'SET key=value'. If you want to include special characters in key, or include semicolon in value, please use quotes, e.g., SET `key`=`value`."
```

Comment:
```suggestion
      "Expected format is 'SET', 'SET key', or 'SET key=value'. If you want to include special characters in key, or include semicolon in value, please use backquotes, e.g., SET `key`=`value`."
```

Reply:
Is the value back ticked as well? That seems weird. Or is the example wrong and it should be quotes for both?

Reply:
Yeah I strongly suspect that the example is incorrect.
Will try running it myself just to be sure, and fix the example.

Reply:
Oh... actually it's not wrong example.

We should backquotes for value when the value contains invalid syntax as below:
```scala
scala> sql("SET a=`1;2;;`").show()
+---+-----+
|key|value|
+---+-----+
|  a|1;2;;|
+---+-----+
```

Otherwise it raises error:
```scala
scala> sql("SET a=1;2;;")
org.apache.spark.sql.catalyst.parser.ParseException:
[INVALID_SET_SYNTAX] Expected format is 'SET', 'SET key', or 'SET key=value'. If you want to include special characters in key, or include semicolon in value, please use quotes, e.g., SET `key`=`value`.(line 1, pos 0)

== SQL ==
SET a=1;2;;
^^^
```